src/cpg_seqr_loader/scripts/alphagenome_var_exp_script.py

```diff
@@ -0,0 +1,652 @@
+window_size = config.config_retrieve(['alphagenome_parm', 'window_size'], default=100)
+scan_span = config.config_retrieve(['alphagenome_parm', 'scan_span'], default=50000)
+plot_non_sig = config.config_retrieve(['alphagenome_parm', 'plot_non_sig'], default=False)
+scan_all_tracks = config.config_retrieve(['alphagenome_parm', 'scan_all_tracks'], default=True)
+epsilon = config.config_retrieve(['alphagenome_parm', 'epsilon'], default=1e-8)
+api_key = config.config_retrieve(['alphagenome_parm', 'api_key'], default=None)
+gtf = config.config_retrieve(
+    ['alphagenome_parm', 'gtf'],
+    default='gs://cpg-common-main/references/alphagenome/gencode.v46.annotation.gtf.gz.feather',
+)
+col_chrom = config.config_retrieve(['alphagenome_parm', 'col_chrom'], default='CHROM')
+col_pos = config.config_retrieve(['alphagenome_parm', 'col_pos'], default='POS')
+col_ref = config.config_retrieve(['alphagenome_parm', 'col_ref'], default='REF')
+col_alt = config.config_retrieve(['alphagenome_parm', 'col_alt'], default='ALT')
+
+
+# precommit check
+
+
+def build_parser() -> argparse.ArgumentParser:
+    P = argparse.ArgumentParser(
+        prog='alpha_variant_scan',
+        description='Scan predicted expression effects of variants across organs using AlphaGenome.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    P.add_argument('--variants', required=True, help='TSV/CSV/VCF of variants.')
+    P.add_argument(
+        '--output-table',
+        default='gs://cpg-kidgen-test/alphagenome_scan_results.csv',
+        help='Output summary table path (csv/tsv/xlsx).',
+    )
+    P.add_argument(
+        '--output-table-sum',
+        default='gs://cpg-kidgen-test/alphagenome_scan_results_variant_organ_summary.csv',
+        help='Output organ summary table path (csv/tsv/xlsx).',
+    )
+    P.add_argument('--output-dir', default='gs://cpg-kidgen-test/alphagenome_scan_plots', help='Dir for plot images.')
+
+    return P
+
+
+# ------------------------------------------------------------------
+# I/O
+# ------------------------------------------------------------------
+
+
+def load_variants_table(path: str, col_chrom: str, col_pos: str, col_ref: str, col_alt: str) -> pd.DataFrame:
+    fp = Path(path)
+    if not fp.exists():
+        raise FileNotFoundError(path)
+    suf = fp.suffix.lower()
+    if suf in {'.tsv', '.txt'}:
+        df = pd.read_csv(fp, sep='\t')
+    elif suf == '.csv':
+        df = pd.read_csv(fp)
+    elif suf == '.vcf':
+        df = _load_vcf(fp)
+    else:
+        # fallback: try tab then comma
+        try:
+            df = pd.read_csv(fp, sep='\t')
+        except Exception:
+            df = pd.read_csv(fp)
+    mapping = {col_chrom: 'CHROM', col_pos: 'POS', col_ref: 'REF', col_alt: 'ALT'}
+    miss = [c for c in mapping if c not in df.columns]
+    if miss:
+        raise ValueError(f'Missing required columns: {miss}')
+    df = df.rename(columns=mapping)[['CHROM', 'POS', 'REF', 'ALT']]
```

Comment:
this whole block can mostly go - we're always supplying TSV data, so need to check. We also don't need alternative names for the CHR/POS/REF/ALT, so the mapping clause can go.

It's not doing any harm, but it's just more code that doesn't help us